content/en/database_monitoring/setup_sql_server/rds.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Setting Up Database Monitoring for SQL Server on Amazon RDS
+kind: documentation
+description: Install and configure Database Monitoring for SQL Server managed on RDS.
+further_reading:
+- link: "/integrations/sqlserver/"
+  tag: "Documentation"
+  text: "Basic SQL Server Integration"
+- link: "/database_monitoring/troubleshooting/?tab=sqlserver"
+  tag: "Documentation"
+  text: "Troubleshoot Common Issues"
+
+
+---
+
+{{< site-region region="us5,gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">Database Monitoring for SQL Server is in private beta. Contact your Customer Success Manager to request access to the beta.</div>
+
+Database Monitoring provides deep visibility into your Microsoft SQL Server databases by exposing query metrics, query samples, explain plans, database states, failovers, and events.
+
+Do the following steps to enable Database Monitoring with your database:
+
+1. [Grant the Agent access to the database](#grant-the-agent-access)
+2. [Install the Agent](#install-the-agent)
+3. [Install the RDS integration](#install-the-rds-integration)
+
+## Before you begin
+
+{{% dbm-sqlserver-before-you-begin %}}
+
+## Grant the Agent access
+
+The Datadog Agent requires read-only access to the database server in order to collect statistics and queries.
+
+Create a read-only login to connect to your server and grant the required permissions:
+
+```SQL
+CREATE LOGIN datadog WITH PASSWORD = '<PASSWORD>';
+CREATE USER datadog FOR LOGIN datadog;
+GRANT CONNECT ANY DATABASE to datadog;
```

Comment:
I'm fairly confident this command fails on RDS, and I think to fix some of the metrics issues I saw I had to create the DD user in every single DB on the host 

there are some details about this in the description of this [jira ticket](https://datadoghq.atlassian.net/browse/DBM-997)

Reply:
Good catch! Fixed https://github.com/DataDog/documentation/pull/13478/commits/a74987759ec4f72134473851135c508ebe903a83